ipmiMgrApp/Db/afcv3.substitutions

```diff
@@ -27,6 +27,12 @@
 #  19   Compact  0xf0    0xc1  0x69  0x10              HS 013 AMC9
 # ------------------------------------------------------------------
 
+file "power_mngr.db"
+{
+    pattern { id    , power_channel }
+            { $(id) , $(power_channel) }
+}
```

Comment:
For a template like this, you can simply do

```
file "power_mngr.db"
{
    {}
}
```

The existing substitutions are already forwarded to templates automatically, so you don't need to specify them. All you need to do is instantiate that file once.

If you prefer to do it like this for documentation purposes though, feel free.